src/content/configuration/dev-server.md

```diff
@@ -749,7 +749,7 @@ module.exports = {
 Usage via the CLI
 
 ```bash
-npx webpack serve --no-live-reload
+npx webpack serve --liveReload
```

Comment:
I notice that it's enabled by default, then how to disable it from CLI?

Reply:
We can not. We can only enable it via cli if disabled in the config.

Reply:
@snitin315 We should support it

Reply:
We support in `webpack-dev-server@beta` but not in v3.11.2

Reply:
I will migrate the docs after rc/major release.